source/JsMaterialX/JsMaterialXCore/JsDocument.cpp

```diff
@@ -83,12 +95,21 @@ extern "C"
             .function("getUnitTypeDef", &mx::Document::getUnitTypeDef)
             .function("getUnitTypeDefs", &mx::Document::getUnitTypeDefs)
             .function("removeUnitTypeDef", &mx::Document::removeUnitTypeDef)
+            .function("getVersionIntegers", ems::optional_override([](mx::Document &self) {
+                auto version = mx::getVersionIntegers();
```

Comment:
This is calling the wrong function. It should invoke `self.getVersionIntegers()`!

Reply:
And the pointer indices below need to be updated accordingly, since `Node::getVersionIntegers` only returns a pair of two ints. Ideally, this is one of the functions that we test explicitly.

Reply:
I think we should make it explicit as you mentioned. Pointer arithmetic for a pair is also not needed.